### ISSUE TITLE
package.json repo is wrong (goes to missing repo)

### ISSUE THREAD

**Issue by srl295** [apache/couchdb-fauxton#1017]
## Expected Behavior

expect repo metadata to go to a valid repository but it goes to https://git-wip-us.apache.org/repos/asf?p=couchdb.git

## Current Behavior

package.json points to https://git-wip-us.apache.org/repos/asf?p=couchdb.git

## Possible Solution

fix package.json to github.com/apache/couchdb-fauxton.git

## Steps to Reproduce (for bugs)

- go to https://www.npmjs.com/package/fauxton 
- click on the repository URL on the right sidebar
- enjoy `404 - No such project `

## Context

- i wanted to see how to see an example of  Grunt and webpack for a couchapp 

## Your Environment

- a browser

**Comment by srl295** [comment]
specifically:
```json
  "repository": {
    "type": "git",
    "url": "https://git-wip-us.apache.org/repos/asf/couchdb.git"
  },
```

- by the way, I'm not sure why the readme mentions an `upstream` of `git@github.com…` (ssh) when https://github.com/apache/couchdb-fauxton.git should be a fine upstream.

**Comment by Antonio-Maranhao** [comment]
Fixed on #1082
